src/Simplex/Messaging/Agent/NtfSubSupervisor.hs

```diff
@@ -73,56 +73,34 @@ processNtfSub c (connId, cmd) = do
       case a of
         Nothing -> do
           withTokenServer $ \ntfServer -> do
-            case clientNtfCreds of
-              Just ClientNtfCreds {notifierId} -> do
-                let newSub = newNtfSubscription userId connId smpServer (Just notifierId) ntfServer NASKey
-                withStore c $ \db -> createNtfSubscription db newSub $ NSANtf NSACreate
-                lift . void $ getNtfNTFWorker True c ntfServer
```

Comment:
unclear to me how this path could occur, possibly dead code

Reply:
possibly not dead - but reusing for case when creds weren't deleted from queue record - still have to re-create instead of reuse